4.processing_image_based_profiles/merge_features_local.sh

```diff
@@ -53,18 +61,33 @@ for patient in "${patient_array[@]}"; do
     mkdir -p "$(dirname "$patient_log_file")"  # create the patients directory if it doesn't exist
     touch "$patient_log_file"  # create the patient log file if it doesn't exist
     {
-        python "$git_root"/4.processing_image_based_profiles/scripts/5.combining_profiles.py --patient "$patient"
-        python "$git_root"/4.processing_image_based_profiles/scripts/6.annotation.py --patient "$patient"
-        python "$git_root"/4.processing_image_based_profiles/scripts/7.normalization.py --patient "$patient"
-        python "$git_root"/4.processing_image_based_profiles/scripts/8.feature_selection.py --patient "$patient"
-        python "$git_root"/4.processing_image_based_profiles/scripts/9.aggregation.py --patient "$patient"
-        python "$git_root"/4.processing_image_based_profiles/scripts/10.merge_consensus_profiles.py --patient "$patient"
+        python "$git_root"/4.processing_image_based_profiles/scripts/6.combining_profiles.py --patient "$patient"
+        python "$git_root"/4.processing_image_based_profiles/scripts/7.annotation.py --patient "$patient"
+        python "$git_root"/4.processing_image_based_profiles/scripts/8.normalization.py --patient "$patient"
+        python "$git_root"/4.processing_image_based_profiles/scripts/9.feature_selection.py --patient "$patient"
+        python "$git_root"/4.processing_image_based_profiles/scripts/10.aggregation.py --patient "$patient"
+        python "$git_root"/4.processing_image_based_profiles/scripts/11.merge_consensus_profiles.py --patient "$patient"
```

Comment:
Given the consistency of numbering consider using an executor which takes this into account and executes each module in the correct sequence with the arguments. This could also be useful more widely. Wasn't there a tool which this file numbering style stemmed from? Should we implement that somewhere / somehow?